### ISSUE TITLE
[BUG] Mirth Connect Administrator Launcher fails to install -- charset.jar

### ISSUE THREAD

**Issue by jabortell** [nextgenhealthcare/connect#4521]
**Describe the bug**
I received a notification that version 1.2 of the Launcher was available. When attempting to install the Mirth Connect Administrator Launcher, it fails with the error: "C:\Program Files\Mirth Connect\Administrator Launcher\jre\lib\charsets.jar Could not create this file. Shall I try again?" I tried reinstalling 1.1, the previous version on my machine, but it failed with the same error. Now I don't have any Launcher on my machine, because 1.2 uninstalled 1.1.

1. Run the Mirth Connect Administrator Launcher Installer for versions 1.1 or 1.2.

**Expected behavior**
The installer installs the Launcher.

**Actual behavior**
The Installer uninstalls the previous version on my machine, then immediately fails. Now I have no launchers.

**Screenshots**

![image](https://user-images.githubusercontent.com/19395866/116094562-a6843a00-a675-11eb-8192-f9737918432e.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Java Distribution/Version
java version "1.8.0_271"
Java(TM) SE Runtime Environment (build 1.8.0_271-b09)
Java HotSpot(TM) 64-Bit Server VM (build 25.271-b09, mixed mode)
 - Connect Version: 3.9.0

**Workaround(s)**
I tried to launch Mirth Connect from the Mirth Administrator on the web, but that also failed, perhaps for different reasons.

**Comment by cturczynskyj** [comment]
We have tested installing and upgrading in a Windows 10 x64 environment and it completes without errors. The fact that neither 1.1, nor 1.2 can install is suspicious. Are you sure you have permission where you are attempting to install the launcher?

**Comment by jabortell** [comment]
Yes, I'm sure. On my machine I am not able to run the installer without running it with administrator privileges.

**Comment by matthydehoog** [comment]
I noticed sometimed the launcher.exe is still running as a (taskmanage )proces. Just closing this launcher solves installation problems.
